### DISCUSSION TITLE
type error expected 0-1 arguments but got two  "next-auth": "^5.0.0-beta.18",

### DISCUSSION THREAD

**Discussion by trondstromlie** [nextauthjs/next-auth#11025] [Help]
Hello not sure if this is a bug or i@m doing something wrong, everything is working but the typescheck is killing my build 

../../node_modules/@auth/core/src/errors.ts:57:24
Type error: Expected 0-1 arguments, but got 2.

  55 |   ) {
  56 |     if (message instanceof Error) {
> 57 |       super(undefined, {
     |                        ^
  58 |         cause: { err: message, ...(message.cause as any), ...errorOptions },
  59 |       })
  60 |     } else if (typeof message === "string") {

i´m running this is in nx monorepo

**Comment by k-taro56** [comment]
It appears that you are using a version of Node.js lower than 16.

The constructor of the Error class must be 16.9.0 or higher as shown below.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause#browser_compatibility

Please update your Node.js as we support Node.js version 18 and above.

We also strongly recommend that you update your Node.js 16 as it is no longer supported.
https://nodejs.org/en/about/previous-releases
